src/utils/build_handlers.js

```diff
@@ -24,12 +29,36 @@ export default function buildHandlers (handlersObj, index) {
           });
       };
 
-      return handlersObj[key];
+      return object;
     } else {
       throw new Error('Intent handler must either be a function or an object ' +
       'with key of "answerWith" which is a function.');
     }
+  }, {});
+
+  delete result.STATE_STRING;
```

Comment:
Something is confusing here with `STATE_STRING`. It is a variable but also here you delete the key that is named `STATE_STRING` right?